### ISSUE TITLE
Legend of Zelda OoT Menu Scrolling Slow

### ISSUE THREAD

**Issue by shinra358** [gonetz/GLideN64#408]
In TLoZOoT, when going to the start menu and pressing Z or R to scroll to the next menu page, it becomes very sluggish or slow when doing so. Does this in P641.6 and in the newer 2.2 revisions. This didn't happen in the old glide.

**Comment by gonetz** [comment]
Can't reproduce that. Screen transition works smooth for me.

**Comment by shinra358** [comment]
Uploading a vid of the settings I have exactly and demonstrating the problem. One moment.

**Comment by Predator82Germany** [comment]
What´s without hi-res or alpha & crc off?

**Comment by shinra358** [comment]
I'm sorry, can you rephrase tha?

**Comment by Predator82Germany** [comment]
Texture pack off

**Comment by shinra358** [comment]
Okay, with texture pack off, it is smooth again.

**Comment by gonetz** [comment]
Regarding texture cache options:

Texture cache on Video tab tells the plugin how much of video memory it can take for textures. Plugin eats lots of video memory when Frame buffer emulation is enabled and twice as much when FSAA enabled.

Texture cache in Textures tab tells the plugin how much of conventional RAM it can take for filtered/enhanced textures. That cache keeps all already loaded game's textures - load once, get instantly. So, it's better to make it large enough.

**Comment by purplemarshmallow** [comment]
Glide64 has a setting called "VRAM size". Is that the same as "Texture cache size" in the video tab? Maybe it should be renamed. Because it gets confused easily with "Texture cache" in the "Textures" tab.

**Comment by shinra358** [comment]
Found out that title of thread wasn't solved but at least the cache thing was. Turns out it is the FSAA that is doing it or assisted in doing it further. Even at 1x. So I turn that off and use FXAA through reshade instead and it is smooth.

**Comment by gonetz** [comment]
Yes, two "Texture cache" options are confusing. I intended to rename one of it before release but forgot about it. Could you suggest a good name for that option?  "VRAM size" is not quite right. Texture cache option controls amount of video memory available for texture allocation. If you set it equal to your video card video memory, you may get performance drop, because video memory is needed not only for textures.

**Comment by Predator82Germany** [comment]
May:
GFX-Memory

**Comment by gonetz** [comment]
Not better than "VRAM size"

**Comment by Predator82Germany** [comment]
Graphic-Space? :)

**Comment by gonetz** [comment]
The name should reflect the fact that that amount of memory is reserved for textures.

**Comment by Predator82Germany** [comment]
TextureOnlySpace?

**Comment by purplemarshmallow** [comment]
Maybe name one setting "VRAM texture cache"
And the other one "cache for enhanced textures"

**Comment by gonetz** [comment]
"VRAM texture cache" is good.
 "enhanced textures cache size" ? may be.

**Comment by purplemarshmallow** [comment]
I think that's good
The name should make clear that this setting is for enhanced textures only although the tooltip explains it so there's a difference to the other setting
